### ISSUE TITLE
enable MiMa post-release

### ISSUE THREAD

**Issue by SethTisue** [scala/scala-library-next#15]

**Comment by SethTisue** [comment]
@julienrf what's the grounds for closing this?

**Comment by julienrf** [comment]
We now have sbt-scala-module 2.4.0, which runs MiMa as part of the tests, and which will be enabled as soon as `versionPolicyIntention` becomes different from `Compatibility.None`. Should we keep the issue open until we perform the first release?

**Comment by SethTisue** [comment]
Yeah, I think so. The new situation is basically the same as the old situation: we need to remember take action after the next release.
